signup.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 회원가입</title>
+    <link rel="icon" href="images/logo/favicon.ico" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="styles/global.css" />
+    <link rel="stylesheet" href="styles/auth.css" />
+    <style>
+      .error-message {
+        color: red;
+        font-size: 0.8em;
+        margin-top: 5px;
+      }
+      .input-item.error input {
+        border-color: red;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="auth-container">
+      <a href="/" class="logo-home-button"
+        ><img src="images/logo/logo.svg" alt="판다마켓 홈"
+      /></a>
+
+      <form method="post" id="signup-form">
+        <div class="input-item">
+          <label for="email">이메일</label>
+          <input
+            id="email"
+            name="email"
+            type="email"
+            placeholder="이메일을 입력해 주세요"
+          />
+          <div id="email-error-message" class="error-message"></div>
+        </div>
+        <div class="input-item">
+          <label for="nickname">닉네임</label>
+          <input
+            id="nickname"
+            name="nickname"
+            type="text"
+            placeholder="닉네임을 입력해 주세요"
+          />
+          <div id="nickname-error-message" class="error-message"></div>
+        </div>
+        <div class="input-item">
+          <label for="password">비밀번호</label>
+          <div class="input-wrapper">
+            <input
+              id="password"
+              name="password"
+              type="password"
+              placeholder="비밀번호를 입력해 주세요"
+            />
+            <img
+              src="images/icons/eye-invisible.svg"
+              alt="비밀번호 숨김"
+              class="toggle-password"
+              id="password-toggle"
+            />
+          </div>
+          <div id="password-error-message" class="error-message"></div>
+        </div>
+        <div class="input-item">
+          <label for="passwordConfirmation">비밀번호 확인</label>
+          <div class="input-wrapper">
+            <input
+              id="passwordConfirmation"
+              name="passwordConfirmation"
+              type="password"
+              placeholder="비밀번호를 다시 한 번 입력해 주세요"
+            />
+            <img
+              src="images/icons/eye-invisible.svg"
+              alt="비밀번호 숨김"
+              class="toggle-password"
+              id="passwordConfirmation-toggle"
+            />
+          </div>
+          <div
+            id="passwordConfirmation-error-message"
```

Comment:
camelCase와 kebab-case의 형태가 모두 보이는데 하나로 통일해서 사용해 주세요!
```suggestion
              id="password-confirmation-toggle"
            />
          </div>
          <div
            id="password-confirmation-error-message"
```